src/Spryker/Zed/EventBehavior/Business/Model/EventResourceQueryContainerPluginIterator.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\EventBehavior\Business\Model;
+
+use Iterator;
+use Propel\Runtime\ActiveQuery\ModelCriteria;
+use Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourceQueryContainerPluginInterface;
+
+class EventResourceQueryContainerPluginIterator implements Iterator
```

Comment:
Please apply same fixes for `sync:data`in `spryker/spryker` repository

Reply:
Done already :)